src/Compilers/Core/VBCSCompilerTests/CompilerServerTests.cs

```diff
@@ -2207,5 +2254,195 @@ public void ExecuteVbcBuildTaskWithServer()
             Assert.Equal(0, result.ExitCode);
             Assert.Equal("Hello from VB", result.Output.Trim());
         }
+
+        // A dictionary with name and contents of all the files we want to create for the ReportAnalyzerMSBuild test.
+        private Dictionary<string, string> ReportAnalyzerMsBuildFiles => new Dictionary<string, string> {
```

Comment:
this is the test that runs compiler through msbuild which consumes target files.